### PR TITLE
#3 add variables to link icons css and fix mixin

### PR DESCRIPTION
**Pull request by evalopezm**
Could you check if `@mixin itemBordersDesign()` is used properly ? 
I've also changed the declaration of the variable in the mixin because it wasn't being recognised.
thanks!

### CODE REVIEW

File: _src/assets/scss/components/_showroom.scss [hunk 1]
```diff
@@ -65,10 +65,7 @@
 }
 
 .media-icon {
-  width: 32px;
-  height: 32px;
-  border: 2px solid $formBorder;
-  border-radius: 50px;
+  @include itemBordersDesign();
```
Comment (Cristinapontijas) [inline _src/assets/scss/components/_showroom.scss]: Good improvement :)

File: _src/assets/scss/core/_mixins.scss [hunk 2]
```diff
@@ -14,7 +14,7 @@
   width: 32px;
   height: 32px;
   border-radius: 16px;
-  border: solid 2px var($formBorder);
+  border: solid 2px $formBorder;
```
Comment (Cristinapontijas) [inline _src/assets/scss/core/_mixins.scss]: are theese variables included in variables scss? I'm not able to see it

**Comment by Cristinapontijas** [comment]
hi Eva! The improvements seem right to me. The only thing I see is the name of the issue, as you are naming this issue `#3 Add Variables...` but it referred to the actual `issue #4`. This may cause a little confusion.
The rest looks fine, thank you!

File: _src/assets/scss/core/_mixins.scss (hunk 2 above)
Comment (evalopezm) [reply _src/assets/scss/core/_mixins.scss]: Yes, they are under borders comment (line 27 on _variables.scss). I thought you had created it.

**Comment by evalopezm** [comment]
Thanks! Perfect work! I've seen you have merged to master. Shouldn't we merge to dev? do we merge to master directly?

**Comment by Cristinapontijas** [comment]
Hello eva!
Sorry I didn't see that it would merge to master. I completely agree with you that it is better to merge to dev, before merging the final result into master. However, when we create a pull request we have to say from where into where you are making the pull request :) So I don't know if this is something you can change when clicking on merge, I'd rather say it is something we must define when creating the pull request.

**Pull request merged**